### DISCUSSION TITLE
Tachiyomi End of Life effects on Taiga?

### DISCUSSION THREAD

**Discussion by sinnx3** [erengy/taiga#1169] [Questions] [answered]
Given Tachiyomi releasing an end of life announcement with their program will this overall affect Taiga as a whole?

Will development continue or is Taiga now End of Life too?

source: [https://tachiyomi.org/news/2024-01-13-goodbye](url)

**Comment by wopian** [comment]
Tachiyomi is completely unrelated to Taiga

**Comment by erengy** [comment, marked as answer]
As @wopian said, Tachiyomi and Taiga are unrelated. They both just happen to be open-source applications with a media tracking feature. There hasn't been any legal issues with Taiga so far, apart from #267 and erengy/taiga-moe#2.

I haven't used Tachiyomi before, but from what I gather, it's been the most convenient way to read manga for some time. While it's sad and infuriating to see a project ending like this due to corporate legal pressure, I understand the contributors' decision. This is one of the reasons why I never wanted to support unofficial streaming services etc. in Taiga, even as an extension (e.g. #184, #254, #702).
